.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         os: [ubuntu-18.04]
         python-version: [3.8]
         geomstats-backend : ['autograd','numpy','pytorch','tensorflow']
-        test-folder : ['tests/tests_geomstats/' , 'tests/test_notebooks.py']
+        test-folder : ['tests/tests_geomstats/' , 'scripts']
```

Comment:
maybe test-folder should be renamed, since "scripts" is not a folder?

Reply:
ah your comment made me realize that the better way to go may be to create a folder `test_scripts` and move there both test files, so that the workflow code becomes much simpler. What do you think?